tests/e2e/gc_cns_nodevm_attachment.go

```diff
@@ -296,8 +296,8 @@ var _ = ginkgo.Describe("[csi-guest] CnsNodeVmAttachment persistence", func() {
 		}()
 
 		ginkgo.By("Create a Pod with PVC created in previous step mounted as a volume")
-
 		pod := fpod.MakePod(namespace, nil, []*v1.PersistentVolumeClaim{pvc}, false, "")
+		pod.Spec.Containers[0].Image = busyBoxImageOnGcr
 		pod, err = client.CoreV1().Pods(namespace).Create(ctx, pod, metav1.CreateOptions{})
```

Comment:
Can we use `createPod` here instead? This will avoid having to touch many files if we need to change image for pods.

Reply:
Why I didn't use `createPod` here becasue we need to validate the pod is in the container creating state, in `createPod` method we are waiting till the pod reaches the running state.